src/server/worker/api_server.go

```diff
@@ -524,7 +524,11 @@ func (a *APIServer) runUserCode(ctx context.Context, logger *taggedLogger, envir
 	// cmd.Process.Wait() then cmd.Wait() will produce an error. So instead we
 	// close the IO using this helper
 	err = cmd.WaitIO(state, err)
-	if err != nil {
+	// We ignore broken pipe errors, these occur very occasionally if a user
+	// specifies Stdin but their process doesn't actually read everything from
+	// Stdin. This is a failure common thing to do, bash by default ignores
```

Comment:
This should probably be re-worded.

Reply:
Good catch, this is one of my classic moves right here, just substituting a completely different word with a few common letters >.<